index.html

```diff
@@ -239,6 +239,46 @@ <h3>Skills</h3>
         </section>
     </main>
 
+    <section id="contactFormBox">
+        <header>
+            <h2 id="getStarted">Get started</h2>
+            <p id="cfp">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna.
+            </p>
+        </header>
+        <form action="https://formspree.io/f/xjvdryln" method="POST" id="contactForm">
+            <input class="formInput" type="text" placeholder="Your name" maxlength="30" name="Your Name" id="yourName" required>
+            <input class="formInput" type="email" placeholder="Your email" name="Your email" id="yourEmail"
+                pattern="[a-zA-Z0-9_]+([.][a-zA-Z0-9_]+)*@[a-zA-Z0-9_]+([.][a-zA-Z0-9_]+)*[.][a-zA-Z]{1,5}"
+                required>
+            <textarea id="bigInput" type="text" placeholder="Enter your message here" maxlength="500" name="Message"
+                cols="30" rows="10" required></textarea>
+            <button type="submit" class="submit-button">Start collaboration</button>
+        </form>
```

Comment:
Well done on the form 💯 but there's something quite not right with the design, your design seems to not be centered, leaning a bit towards the left side of the screen. This issue seems to also be present in the rest of the screen but this is about the contact form. Take a look 👇🏽 

## Your design
![Screenshot 2023-02-28 at 20 12 38](https://user-images.githubusercontent.com/45739963/222019475-23e246a2-5b7c-46ef-a4b4-a729c0e58126.png)

## Figma
![Screenshot 2023-02-28 at 20 14 13](https://user-images.githubusercontent.com/45739963/222019662-754c1a6e-1318-48c0-82c8-fdc1b09fa016.png)